### PR TITLE
README.rst: update wrt the repo archiving

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Implements https://forum.ansible.com/t/reduce-fragmentation-moving-stuff-to-the-forum-and-archiving-repos/5141/2?u=andersson007

Another related PR is https://github.com/ansible/ansible-documentation/pull/1529

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @felixfontein

File: README.rst [hunk 1]
```diff
@@ -2,278 +2,6 @@
...
-Please note that Ansible 2.9 no longer receives bugfixes. Only security issues can be fixed, and eventually it will be end of life with no more fixes accepted. See `ansible-core release cycle <https://docs.ansible.com/ansible/devel/reference_appendices/release_and_maintenance.html#ansible-core-release-cycle>`_ for whether 2.9 is still accepting security fixes or not.
-
-The process for fixing a security issue is as follows:
-
-1. PR for bug fix made against the Collection
-2. PR gets merged into Collection
-3. Raise PR directly against ``ansible/ansible:stable-2.9`` (ie not a backport) including a ``changelogs/fragments/`` file
-
-The changes in the PR against ``ansible/ansible:stable-2.9`` should be as close as possible to the changes in the collection original PR, and you should add a reference to the collection PR in the ``ansible/ansible`` PR.
+If you are here to discuss Ansible community collections or request a repository for a community collection under the ``ansible-collections`` GitHub organization, see the [communication guide](https://docs.ansible.com/ansible/devel/community/communication.html).
```
Comment (mariolenz) [inline README.rst:7]:
```suggestion
If you are here to discuss Ansible community collections or request a repository for a community collection under the ``ansible-collections`` GitHub organization, see the `communication guide <https://docs.ansible.com/ansible/devel/community/communication.html>`_.
```

This looks weird at the moment:

```
see the [communication guide](https://docs.ansible.com/ansible/devel/community/communication.html)
```

I'm not 100% sure but I hope my suggestion will create a link.

Comment (felixfontein) [reply README.rst:7]: Yes, your suggestion is correct.

Comment (Andersson007) [reply README.rst]: oh, it's .rst, right, thanks! committed, PTAL

**Review by mariolenz — APPROVED** [review]
LGTM

**Comment by Andersson007** [comment]
@mariolenz @felixfontein thanks for reviewing!

**Pull request merged**